tensorboard/webapp/experiments/types.ts

```diff
@@ -22,6 +22,9 @@ export declare interface Experiment {
   hparams?: string;
   tags?: string[];
   related_links?: Array<{name: string; url: string}>;
+  // These state values were chosen to follow these AIPs
```

Comment:
That's not entirely accurate. 'hidden' doesn't appear in either AIP:

These state values were chosen to follow these AIPs https://google.aip.dev/164 and https://google.aip.dev/216 but with 'delete' replaced by 'hidden'.

It would be worth asking NickG if he'd consider renaming 'hidden' to 'delete' given 'delete' is the term used by the "Soft delete" AIP.

Reply:
@groszewn thoughts?

Reply:
We specifically don't have a "delete" option because the underlying storage system doesn't currently support a deleted state right now. I drafted an internal design document around introducing a deleted state, but it would be on top of the current values and not a replacement for "hidden".